src/main/java/our/member/domain/TokenEntity.java

```diff
@@ -0,0 +1,62 @@
+package our.member.domain;
+
+import java.time.LocalDateTime;
+
+public class TokenEntity {
```

Comment:
`TokenEntity` 도 좋은 네이밍이지만, `Token`이라 이름을 지어도 도메인 패키지 아래에 있어서 엔티티라 알 수 있을 것 같아요!
```suggestion
public class Token {
```